WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -426,6 +426,24 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
  */
 + (NSString *)userInterfaceStyle
 {
+#if !TARGET_OS_SIMULATOR
```

Comment:
I assume it should also work on simulators

Reply:
fb_getAppearance itself worked, but the result was wrong on my local sometimes. I forgot exact combination though...
For example, even i set a simulator as dark mode, it returned as light.

Reply:
This worked only iOS 15+, so not simulators/real devices